use-shopping-cart/package.json

```diff
@@ -25,7 +25,7 @@
     "dev": "ntl dev"
   },
   "peerDependencies": {
-    "react": "16.13.1"
+    "react": "^16.13.1 || ^17.0.1"
```

Comment:
```suggestion
    "react": "^16"
```

Reply:
@ChrisBrownie55 `^16.8.0` only matches 16, not 17.

Reply:
My bad changed it to >=